shared/login/relogin/container.tsx

```diff
@@ -51,9 +47,9 @@ class LoginWrapper extends React.Component<Props, State> {
   }
 
   componentDidUpdate(prevProps: Props) {
-    // Clear the password when there's an input error.
-    if (this.props.inputError !== prevProps.inputError) {
-      this.setState(p => ({inputKey: p.inputKey + 1, password: ''}))
+    // Clear the password when there's an error.
+    if (this.props.error !== prevProps.error) {
+      this.setState(_ => ({password: ''}))
```

Comment:
callback form is only used if you need the old value, just `this.setState({password: ''})`